onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -321,9 +321,10 @@ static bool IsQDQGraph(const onnxruntime::GraphViewer& graph_viewer) {
   return false;
 }
 
-static void DumpOpenVINOEPModel(const std::filesystem::path& onnx_model_path_name,
-                                ONNX_NAMESPACE::ModelProto* model_proto,
-                                const onnxruntime::Node& fused_node) {
+static void DumpOpenVINOEPModel([[maybe_unused]] const std::filesystem::path& onnx_model_path_name,
+                                [[maybe_unused]] ONNX_NAMESPACE::ModelProto* model_proto,
+                                [[maybe_unused]] const onnxruntime::Node& fused_node) {
+#ifdef  NOT_RELEASE
```

Comment:
Can we rename this definition NOT_RELEASE 

Lets have a macro for RELEASE 
target_compile_options(onnxruntime_providers_openvino PRIVATE
    $<$<CONFIG:Release>>:-DRELEASE>
  )
  
  #ifndef RELEASE